samples/SQL/allowed-sql-database-collation/azurepolicy.json

```diff
@@ -0,0 +1,46 @@
+{
+   "properties": {
+     "displayName": "allowed-sql-database-collation",
+     "policyType": "Custom",
+     "mode": "All",
+     "metadata": {
+       "category": "SQL"      
+     },
+     "parameters": {
+       "listOfAllowedCollations": {
+         "type": "Array",
+         "metadata": {
+           "displayName": "Allowed Collation Names",
+           "description": "A list of approved collations"
```

Comment:
Give an example collation name so users can be assured that the naming scheme they are using is expected